1_Diversity-Participation.md

```diff
@@ -4,7 +4,9 @@ Activity Metric | Description
 --- | ---
 New Contributors* vs Maintainers** | Ratio of new contributors to maintainers over time
 Contributor Demographics | Gender, age, location, education, and skills over time
-New Contributor Corporate Affiliations | New corporate affiliations over time
+Leadership Demographics | Demographics of project's leadership (e.g. Board, Technical Steering Committee, Maintainers, etc.) over time
+New Contributor Organizations | New organizations contributing to the project over time

```

Comment:
Do we need to have separate metrics for new organizations vs. the number of contributing organizations? They seem a bit redundant to me because if the number of contributing organizations increases by X over a specific time period, you can assume there are X new contributing organizations.

Reply:
I don't think it hurts to keep separate metrics as the "time period" in question may be different.  For example, one may want to know how many new organizations joined the project over the past 12 months while someone else may ask how many total organizations participated in the project since it started in 2015 (for example). 

Also, some organizations may stop contributing and the number of organizations could decrease depending on the time period chosen....

Reply:
I see @BenLloydPearson 's point about redundancy and I see the value in tracking the new and total separate as @rpaik  explains.  
In the data, both metrics may be informed by the same information. Explicating them separately in our metric definition accounts for the questions and metrics people ask of the data. -- Keeping both is my vote.